app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -168,10 +168,10 @@ def after_destroy(_id)
         # leaving id to avoid changing the method's parameters prior to release
         respond_to do |format|
           format.html do
-            redirect_to my_collections_path,
+            redirect_to hyrax.my_collections_path,
```

Comment:
This was needed to fix the controller test that was checking paths being redirected to.

Reply:
any idea why this popped up now?